api/BarProp.json

```diff
@@ -48,6 +48,54 @@
           "deprecated": false
         }
       },
+      "is_popup": {
```

Comment:
I can't find this in the IDL for the `BarProp` interface, which makes me think this is not actually exposed to web developers directly. Instead, it seems to change the value of `visible`. Is that right?

If it is, I think the right way to go is to characterize this is as a change in the way `visible` works, rather than introducing a new feature entry here. I'm not sure how best to represent this change in Firefox 96, however. I think the core question is: before Firefox 96, what did `visible` return and what does it return from 96?

Reply:
yes, the change is the value of `visible`.

https://html.spec.whatwg.org/multipage/window-object.html#dom-barprop-visible

The change is that:
  * before Firefox 96, `BarProp.value` was returning the actual visibility of each UI part (toolbar, menubar, etc), with exceptions that `scrollbars.visible` is always true
  * after Firefox 96, `BarProp.value` returns the negation of "is popup" of the top-level browsing context, regardless of the actual visibility of each UI parts

"is popup" of the top-level browsing context defaults to false, and is calculated from `features` parameter of `window.open`.


currently other browsers return different value for each, for `BarProp.value`.
Safari also returns the actual visibility of each UI parts, but with yet another exceptions.
Chrome returns whether each UI parts are enabled in `features` parameters of `window.open`, regardless of the actual UI parts visibility.